test/plots/piePlotTests.ts

```diff
@@ -187,7 +187,8 @@ describe("Plots", () => {
           { value: 5000 },
           { value: 5000 }];
         let dataset = new Plottable.Dataset(data);
-        piePlot.addDataset(dataset).outerRadius(500);
+        piePlot.addDataset(dataset);
+        piePlot.outerRadius(500);
```

Comment:
For javascript no, for typescript yes even though the intention through typescript is visible.  I'd like to say that it's not an API breaking change, but we can do a quick survey.